src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: $cell-size;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin: 50px 48px 46px 0;
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+  &:nth-last-child(-n + 4) {
+    margin-top: 0;
+    margin-bottom: 78px;
+  }
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    margin-bottom: 40px;
+  }
+  &__title {
+    font-weight: 500;
+    margin: 0;
```

Comment:
is some tags have default margins, you can reset it in reset.scss for the whole tag